packages/@lwc/features/src/__tests__/flags.spec.ts

```diff
@@ -139,6 +163,29 @@ const nonProdTests = {
               if (runtimeFlags.ENABLE_FEATURE_TRUE) {
                 console.log('this looks like a bad idea');
               }
+
+              if (!runtimeFlags.ENABLE_FEATURE_TRUE) {
+                console.log('this looks like a bad idea');
+              }
+            }
+        `,
+    },
+    'should throw an error if the flag is undefined': {
+        error: 'Invalid feature flag "ENABLE_THE_BEER". Flag is undefined.',
+        code: `
+            import featureFlags from '@lwc/features';
+            if (featureFlags.ENABLE_THE_BEER) {
+                console.log('featureFlags.ENABLE_THE_BEER');
+            }
+        `,
+    },
+    'should throw an error if the flag name is formatted incorrectly': {
+        error:
+            'Invalid feature flag "enable_the_beer". Flag name must only be composed of uppercase letters and underscores.',
```

Comment:
can the flag be alpha numeric upper case chars?

Reply:
No numbers for now. We can add that down the line if it ever becomes necessary.